components/cam/cime_config/config_component.xml

```diff
@@ -363,8 +363,10 @@
     <desc compset="_CAM5%MG1M"   >MG1.0 w/ modified activation:</desc>
     <desc compset="_CAM5%UNI"    >UNICON (modified mg1.0):</desc>
     <desc compset="CAM[45].*_BGC%B">CAM prognostic CO2 cycle turned on.</desc>
-    <desc compset="_CAM5.*SCREAM-LR"   >Low Resolution Fortran version of SCREAM with SHOC, P3, and RRTMG.  All other aspects of this run are identical to E3SMv1</desc>
-    <desc compset="_CAM5.*SCREAM-HR"   >Fortran version of SCREAM with NH dycore, SHOC, P3, RRTMGP, and no deep convection scheme called.</desc> 
+    <desc compset="_CAM5.*SCREAM-LR-DYAMOND1">Fortran version of SCREAM with SHOC, P3, and RRTMGP. Initialized for DYAMOND1 (2016-08-01).</desc>
```

Comment:
Maybe we should note in each desc for all SCREAM compsets that prescribed aerosols are used?

Reply:
Oh yeah, good idea. I'll push a commit.

Reply:
Done.